1506632784021	Sync.Service	DEBUG	User-Agent: Firefox/55.0.3 (Linux x86_64) FxSync/1.57.0.20170918122929.desktop
1506632784021	Sync.Service	INFO	Starting sync at 2017-09-28 17:06:24 in browser session __FA3havkW2t
1506632784021	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1506632784022	Sync.Telemetry	TRACE	observed weave:service:sync:start 
1506632784023	Sync.Status	INFO	Resetting Status.
1506632784023	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1506632784024	Sync.BrowserIDManager	DEBUG	_ensureValidToken already has one
1506632784042	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
1506632874348	Sync.AsyncResource	WARN	GET request to https://sync-538-us-west-2.sync.services.mozilla.com/1.5/81964526/info/collections failed: [Exception... "NS_ERROR_NET_TIMEOUT"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "JS frame :: resource://services-sync/resource.js :: Channel_onStopRequest :: line 422"  data: no] Stack trace: Channel_onStopRequest()@resource://services-sync/resource.js:422 < waitForSyncCallback()@resource://services-common/async.js:98 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:168 < promiseSpinningly()@resource://services-common/async.js:234 < _fetchInfo()@resource://services-sync/service.js:470 < sync()@resource://services-sync/stages/enginesync.js:79 < onNotify()@resource://services-sync/service.js:1081 < WrappedNotify()@resource://services-sync/util.js:160 < WrappedLock()@resource://services-sync/util.js:116 < _lockedSync()@resource://services-sync/service.js:1071 < sync/<()@resource://services-sync/service.js:1063 < WrappedCatch()@resource://services-sync/util.js:91 < sync()@resource://services-sync/service.js:1052
1506632874351	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1506632874351	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1506632874353	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms.
1506632874354	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "NS_ERROR_NET_TIMEOUT"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "JS frame :: resource://services-sync/resource.js :: Channel_onStopRequest :: line 422"  data: no] Stack trace: Channel_onStopRequest()@resource://services-sync/resource.js:422 < waitForSyncCallback()@resource://services-common/async.js:98 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:168 < promiseSpinningly()@resource://services-common/async.js:234 < _fetchInfo()@resource://services-sync/service.js:470 < sync()@resource://services-sync/stages/enginesync.js:79 < onNotify()@resource://services-sync/service.js:1081 < WrappedNotify()@resource://services-sync/util.js:160 < WrappedLock()@resource://services-sync/util.js:116 < _lockedSync()@resource://services-sync/service.js:1071 < sync/<()@resource://services-sync/service.js:1063 < WrappedCatch()@resource://services-sync/util.js:91 < sync()@resource://services-sync/service.js:1052
1506632874357	Sync.Telemetry	TRACE	observed weave:service:sync:error 
1506632874360	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "NS_ERROR_NET_TIMEOUT"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "JS frame :: resource://services-sync/resource.js :: Channel_onStopRequest :: line 422"  data: no] Stack trace: Channel_onStopRequest()@resource://services-sync/resource.js:422 < waitForSyncCallback()@resource://services-common/async.js:98 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:168 < promiseSpinningly()@resource://services-common/async.js:234 < _fetchInfo()@resource://services-sync/service.js:470 < sync()@resource://services-sync/stages/enginesync.js:79 < onNotify()@resource://services-sync/service.js:1081 < WrappedNotify()@resource://services-sync/util.js:160 < WrappedLock()@resource://services-sync/util.js:116 < _lockedSync()@resource://services-sync/service.js:1071 < sync/<()@resource://services-sync/service.js:1063 < WrappedCatch()@resource://services-sync/util.js:91 < sync()@resource://services-sync/service.js:1052
1506632874364	Sync.ErrorHandler	DEBUG	Addons installed: 7
1506632874365	Sync.ErrorHandler	DEBUG	 - Tab Groups, version 2.1.4, id tabgroups@quicksaver
1506632874365	Sync.ErrorHandler	DEBUG	 - Restart, version 3.0.2, id Restart@schuzak.jp
1506632874365	Sync.ErrorHandler	DEBUG	 - Violentmonkey, version 2.6.3, id {aecec67f-0d10-4fa7-b7c7-609a2db280cf}
1506632874365	Sync.ErrorHandler	DEBUG	 - LastPass: Free Password Manager, version 4.1.65a, id support@lastpass.com
1506632874365	Sync.ErrorHandler	DEBUG	 - uBlock Origin, version 1.14.10, id uBlock0@raymondhill.net
1506632874365	Sync.ErrorHandler	DEBUG	 - Reddit Enhancement Suite, version 5.8.6, id jid1-xUfzOsOFlzSOXg@jetpack
1506632874365	Sync.ErrorHandler	DEBUG	 - Stylus, version 1.1.4.2, id {7a7a4a92-a2a0-41d1-9fd7-1e92480d612d}
